app/controllers/v0/benefits_claims_controller.rb

```diff
@@ -8,6 +8,8 @@ class BenefitsClaimsController < ApplicationController
     before_action { authorize :lighthouse, :access? }
     service_tag 'claims-shared'
 
+    SUPPRESSED_EVIDENCE_REQUESTS = ['Attorney Fees', 'Secondary Action Required', 'Stage 2 Development'].freeze
```

Comment:
I see this variable repeated over three different files. I worry it could change in one place but not another. Would it make sense to put it one place and call it where it's needed?

Reply:
It would! All this work **should** live in the lighthouse service (as it did in the original work) but this PR temporarily splits it up so as to be deployed separately between mobile and web.

In the interim I'll move this somewhere more centralized, since it might be a little while before I revisit it and move everything back into the service.

Reply:
ok, sounds good!